src/views/start/index.jsx

```diff
@@ -72,9 +73,30 @@ export default class Start extends Component {
   };
 
   changeSorting = (fileStats, attribute) => {
-    this.sortByAttribute(fileStats, 'file');
-    this.sortByAttribute(fileStats, attribute);
-    this.setState({ fileStats });
+    // By default, sort by file path ascendingly
+    this.sortByAttribute(fileStats, 'file', 'asc');
+
+    let [orderByAttribute, order] = this.state.orderBy.split(' ');
+    if (orderByAttribute === attribute) {
+      switch (order) {
+        case 'asc':
+          order = 'desc';
+          break;
+        case 'desc':
+          order = 'asc';
+          break;
+        default:
+          order = 'desc';
+      }
+    } else {
+      order = 'desc';
+    }
+
+    this.sortByAttribute(fileStats, attribute, order);
+    this.setState({
+      fileStats,
+      orderBy: `${attribute} ${order}`
```

Comment:
If the attribute includes a space the asc sorting will not work. I would prefer using an object with two keys e.g. "orderedAttribute" and "order" to prevent this from happening